doc/releases/changelog-0.35.0.md

```diff
@@ -70,27 +210,38 @@
     >>> X(0)
     X(0)
     ```
+    
+    The original long-form names `Identity`, `PauliX`, `PauliY`, and `PauliZ` remain available, but
+    use of the short-form names is now recommended.
+
+  * A new `qml.commutator` function is now available that allows you to compute commutators between
+    PennyLane operators.
+    [(#5051)](https://github.com/PennyLaneAI/pennylane/pull/5051)
+    [(#5052)](https://github.com/PennyLaneAI/pennylane/pull/5052)
+    [(#5098)](https://github.com/PennyLaneAI/pennylane/pull/5098)
 
-  * PennyLane will try to automatically work with a Pauli representation of operators when
-    available. The Pauli representation can be optionally accessed via `op.pauli_rep`:
+    ```pycon
+    >>> qml.commutator(X(0), Y(0))
+    2j * Z(0)
+    ```
+  
+  * PennyLane can now speed up operator arithmetic by harnessing an internal Pauli representation,
+    if available.
     [(#4989)](https://github.com/PennyLaneAI/pennylane/pull/4989)
     [(#5001)](https://github.com/PennyLaneAI/pennylane/pull/5001)
     [(#5003)](https://github.com/PennyLaneAI/pennylane/pull/5003)
     [(#5017)](https://github.com/PennyLaneAI/pennylane/pull/5017)
     [(#5027)](https://github.com/PennyLaneAI/pennylane/pull/5027)
 
+    The Pauli representation can be optionally accessed via `op.pauli_rep`:
```

Comment:
Not sure I understand this one. How does the speedup get triggered? What is this "internal Pauli representation" and how is it different than a "regular" one (what is that even 🤔 😅 )

Reply:
The speedup is automatic if there is a Pauli representation available.

Every operator $O$ can be represented as a linear combination of tensor products of Pauli operators $P_{\vec{i}} = P_{i_{1}} \otimes P_{i_2} \otimes \ldots \otimes P_{i_n}$ with $P_{i} \in  \\{I, X, Y, Z \\}$, where the coefficients of the linear combination are found via $c_{\vec{i}} = \mbox{Tr}(O P_{\vec{i}})$.

Suppose you have operators $O$, $P$, $Q$ and calculate $O P + Q$. In new op arithmetic, PennyLane monitors the Pauli representation of $O$, $P$, and $Q$ (if available) and uses that to perform the calculation. Moreover, this should hopefully be fast because we're using the efficient representation given by `PauliWord` and `PauliSentence`. In old op arithmetic, I think we were partly in this direction if using `Hamiltonian` objects, but the hope is that things are now more standardised and more efficient.

Reply:
Ah! Ok this is clear to me now.